webhook-service/handler/handler_test.go

```diff
@@ -973,6 +1007,16 @@ func TestTaskHandler_Execute_WebhookConfigInProject(t *testing.T) {
 }
 
 func Test_createRequest(t *testing.T) {
+	templateEngineMock := &fake.ITemplateEngineMock{}
+	secretReaderMock := &fake.ISecretReaderMock{}
+	curlExecutorMock := &fake.ICurlExecutorMock{}
+	curlValidatorMock := &fake.ICurlValidatorMock{}
+	curlValidatorMock.ValidateFunc = func(request lib.Request) error {
+		return nil
+	}
+
+	taskHandler := handler.NewTaskHandler(templateEngineMock, curlExecutorMock, curlValidatorMock, secretReaderMock)
+
 	tests := []struct {
```

Comment:
let's also add a test case where the validator returns an error

Reply:
good catch, added :)